### ISSUE TITLE
Bug (need support RWG)

### ISSUE THREAD

**Issue by Envel-Nikita-Gutsenkov** [TheBusyBiscuit/ExoticGarden#126]
## Description (Required)
At the moment, the game has only one actively developed world generator - RWG and for some reason Slimefun (ExoticGarden) does not support it. The bug is that when you grow a tree from Slimefun, two trees grow in each other: a tree from SlimeFun and a tree from RWG.
Please add support!

API RWG: https://github.com/SyntaxPhoenix/SyntaxApi

RWG PLUGIN: https://www.spigotmc.org/resources/realisticworldgenerator.15905/

## Steps to reproduce the Issue (Required)
growing a tree from ExoticGarden: https://imgur.com/a/FFrrkJ0

 - Minecraft Version: all (1.15.2)
 - CS-CoreLib Version: all
 - Slimefun Version: all
 - ExoticGarden Version: all

**Comment by TheBusyBiscuit** [comment]
1. This is not a bug report.
2. The versions "all" do not exist... Without proper versions, a report will be ignored.

RWG is distributed and **sold** by a company.
ExoticGarden meanwhile is open-source, free and has no particular owner or organization behind it, it is community developed.
So I really feel like you should be asking them to add support for it instead of posting it here.
Besides... ExoticGarden is open-source, should they decide that the support should happen inside of this project, they can always add it themselves. Again, it's open-source.

Lastly: The source for RWG is closed, meaning we **cannot** access it.
The API you linked has nothing to do with what you want, that API alone does not allow a compatibility like that to be added.
In other words: Adding it is quite impossible, unless they decide to do it themselves.
